src/AjaxDynamicDataTable.jsx

```diff
@@ -4,15 +4,15 @@ import DynamicDataTable from "./DynamicDataTable";
 
 class AjaxDynamicDataTable extends Component {
 
-    constructor() {
-        super();
+    constructor(props) {
+        super(props);
 
         this.state = {
             rows: [],
             currentPage: 1,
             totalPages: 1,
-            orderByField: null,
-            orderByDirection: null,
+            orderByField: this.props.defaultOrderByField,
+            orderByDirection: this.props.defaultOrderByDirection,
```

Comment:
Use destructuring here.

[Referenced in our styleguide.](https://github.com/langleyfoxall/technologies/blob/master/styleguides/javascript/core.md#destructuring)